### ISSUE TITLE
Selectable Install Options: Download Pax vs Upload Offline Pax vs SMP/E Install

### ISSUE THREAD

**Issue by DivergentEuropeans** [zowe/zen#38]
Zen will need to support the following install options which will change the behavior of the wizard based on the selection:
1) Install Zowe via PAX download (the download process will likely have a separate ticket as it has logistic hurdles)

2) Install via Uploading a Zowe PAX (offline install)

3) SMP/E install

SMP/E likely has other requirements or constraints that the PAX install does not share so the wizard should update according to the selection.

**Comment by jp669844** [comment]
I agree that SMP/E would require a new screen. The question is whether it makes sense to implement SMP/E installation at all, as this is already covered by the z/OSMF PSWI. What could make sense instead is to allow the user to point ZEN to the existing SMP/E environment that was manually created by a sysprog.

Does option 1 also include a license agreement screen before download?
